### PR TITLE
Feature: lint less files with csslint

### LINKED ISSUE #17: Add CSS Linting

### PR DESCRIPTION
**Pull request by jamesjwarren**
Closes #17 

Here is an example of some of the options available in the .cssslintrc file https://github.com/twbs/bootstrap/blob/master/less/.csslintrc

Bootstrap use this in combination with csscomb, so may be worth using that too.

### CODE REVIEW

File: app/less/.csslintrc [hunk 1]
```diff
@@ -0,0 +1,3 @@
+{
+
```
Comment (edwardoparearyee) [inline app/less/.csslintrc]: I don't mind using bootstrap's options here as long as it doesn't conflict with our [styleguide](https://github.com/thisissoon/angular-start/blob/master/app/less/README.md)

File: package.json [hunk 2]
```diff
@@ -24,6 +24,7 @@
     "grunt-contrib-uglify": "~0.9.1",
     "grunt-contrib-watch": "~0.6.1",
     "grunt-contrib-yuidoc": "~0.10.0",
+    "grunt-lesslint": "^1.4.0",
```
Comment (edwardoparearyee) [inline package.json]: should use `~` instead of `^` so as to not introduce breaking changes

**Comment by edwardoparearyee** [comment]
looking good so far :smile:

**Comment by jamesjwarren** [comment]
@edwardoparearyee I've added csslint rules and also configured csscomb with rules from bootstraps example, these are mainly consistent with out styleguide anyway. CSScomb will ensure less rules are formatted consistently and properties are ordered consistently.

**Comment by edwardoparearyee** [comment]
:+1: LGTM

**Pull request merged**